docs/tutorials/quick-start/quick-start-part-1.md

```diff
@@ -0,0 +1,1107 @@
+    </button>
+    <button
+      className={styles.asyncButton}
+      onClick={() => dispatch(incrementAsync(Number(incrementAmount) || 0))}
+    >
+      Add Async
+    </button>
+  </div>
+)
+```
+
+We _could_ keep the current number string in the Redux store, by dispatching an action in the input's `onChange` handler and keeping it in our reducer. But, that doesn't give us any benefit. The only place that text string is used is here, in the `<Counter>` component. (Sure, there's only one other component in this example: `<App>`. But even if we had a larger application with many components, only `<Counter>` cares about this input value.)
+
+So, it makes sense to keep that value in a `useState` hook here in the `<Counter>` component.
+
+Similarly, if we had a boolean flag called `isDropdownOpen`, no other components in the app would care about that - it should really stay local to this component.
+
+**In a React + Redux app, your global state should go in the Redux store, and your local state should stay in React components.**
+
+If you're not sure where to put something, here are some common rules of thumb for determining what kind of data should be put into Redux:
+
+- Do other parts of the application care about this data?
+- Do you need to be able to create further derived data based on this original data?
+- Is the same data being used to drive multiple components?
+- Is there value to you in being able to restore this state to a given point in time (ie, time travel debugging)?
+- Do you want to cache the data (ie, use what's in state if it's already there instead of re-requesting it)?
+- Do you want to keep this data consistent while hot-reloading UI components (which may lose their internal state when swapped)?
+
+This is also a good example of how to think about forms in Redux in general. **Most form state probably shouldn't be kept in Redux.** Instead, keep the data in your form components as you're editing it, and then dispatch Redux actions to update the store when the user is done.
+
+One other thing to note before we move on: remember that `incrementAsync` thunk from `counterSlice.js`? We're using it here in this component. Notice that we use it the same way we dispatch the other normal action creators. This component doesn't care whether we're dispatching a normal action or starting some async logic. It only knows that when you click that button, it dispatches something.
+
+### Providing the Store
+
+We've seen that our components can use the `useSelector` and `useDispatch` hooks to talk to the Redux store. But, since we didn't import the store, how do those hooks know what Redux store to talk to?
+
+Now that we've seen all the different pieces of this application, it's time to circle back to the starting point of this application and see how the last pieces of the puzzle fit together.
+
+```jsx title="src/components/HelloCodeTitle.js"
+function HelloCodeTitle(props) {
+  return <h1>Hello, {props.name}</h1>
+}
+```
```

Comment:
Maybe I'm wrong, but this looks like a random piece of code pasted here by accident :)

```suggestion
```
```
```

Reply:
Good catch, yeah. I think that's left over from me testing out the new code block title feature from Docusaurus, and I just forgot to remove it.